homeassistant/components/media_player/sonos.py

```diff
@@ -42,6 +42,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     players = None
     hosts = config.get('hosts', None)
+    group_devices = config.get('group_devices', False)
```

Comment:
I don't know all details but in my experiments it looks like a player always has a group and if it's alone it's the coordinator for that group too.

Given that there are usually groups and it's only players that are in a group and not the coordinator that are kinda useless (imho, but there might be so I would just keep it as a config option, e.g I don't know how setups with more devices, bridges or soundbars look like since I only have 4 Play:5 to play with) we could also call this `hide_group_members` though `groups_only` might represent the intend of this config a bit better, I'll probably rename it to that :)